app/deliver_grant_funding/routes/reports.py

```diff
@@ -389,6 +391,58 @@ def change_group_display_options(grant_id: UUID, group_id: UUID) -> ResponseRetu
     )
 
 
+@deliver_grant_funding_blueprint.route(
+    "/grant/<uuid:grant_id>/group/<uuid:group_id>/change-add-another-summary", methods=["GET", "POST"]
+)
+@has_deliver_grant_role(RoleEnum.ADMIN)
+@auto_commit_after_request
+def change_group_add_another_summary(grant_id: UUID, group_id: UUID) -> ResponseReturnValue:
+    db_group = get_group_by_id(group_id)
+    form = GroupAddAnotherSummaryForm(
+        available_questions=db_group.cached_questions,
+        questions_to_show_in_add_another_summary=db_group.presentation_options.add_another_summary_line_question_ids
+        if db_group.presentation_options.add_another_summary_line_question_ids
+        else None,
+    )
+
+    if form.validate_on_submit():
+        # TODO using deepcopy here because otherwise the changes don't get flushed to the database
+        # can we make the presentation_options use mutable_json_type or similar so we don't need to do this?
```

Comment:
Hmm yeah ok this seems super frustrating/like a bit of a gotcha 🫠  I think we probably should do something a bit reliable now rather than just drop a TODO.

Small workaround - just move the deepcopy into the interface? So that anything calling it will get the update tracked?

I've had a bit of a play to see if we can combine the pydantic model and the mutable_json_type but to no avail